pkg/controllers/openshift-route-controller-manager.go

```diff
@@ -80,34 +80,20 @@ kubeClientConfig:
     contentType: "application/json"
 servingInfo:
   bindAddress: "0.0.0.0:8445"
-  certFile: ` + cfg.DataDir + `/resources/openshift-controller-manager/secrets/tls.crt
-  keyFile:  ` + cfg.DataDir + `/resources/openshift-controller-manager/secrets/tls.key
+  certFile: ` + cfg.DataDir + `/resources/route-controller-manager/secrets/tls.crt
+  keyFile:  ` + cfg.DataDir + `/resources/route-controller-manager/secrets/tls.key
   clientCA: ` + cryptomaterial.UltimateTrustBundlePath(cryptomaterial.CertsDirectory(cfg.DataDir)) + `
 controllers:
 - "openshift.io/ingress-ip"
 - "openshift.io/ingress-to-route"
```

Comment:
are these configs going to be always hardcoded? Or can a user acccess these via the API?

Reply:
We don't expect to expose these configuration options to the user.